### ISSUE TITLE
Flash on section background image hover and transition time ignored

### ISSUE THREAD

**Issue by yankiara** [elementor/elementor#7989]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

I have set a section with background image and a different background image on hover.
After I load the page, when I hover the section for the first time, there's a flash with global background color before the image shows.

It seems the hover image is loaded only on first hover, and as it takes a little time, the background color appears shortly before the image is displayed.

I've tried on Firefox and Chrome, with preload option on cache plugins like Swift Performance, and the issue is the same.

Besides, with Firefox, transition time is ignored.

## Steps to reproduce

1. Create a section with background image and minimum height like 400px
2. Set a different background image on hover
3. Load the page and hover the section

4. For transition issue, set a 1 second transition and load page with Firefox. Transition value is ignored.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.3.14
	PHP Version: 7.2.16
	PHP Max Input Vars: 2000000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.1.1
	Site URL: http://testelem.yankiara.odns.fr
	Home URL: http://testelem.yankiara.odns.fr
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 4096M
	Permalink Structure: /%postname%/
	Language: fr-FR
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Twenty Nineteen
	Version: 1.3
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: fr_FR
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:66.0) Gecko/20100101 Firefox/66.0

== Active Plugins ==
	Elementor
		Version: 2.5.14
		Author: Elementor.com

**Comment by KappaStel** [comment]
There is a workaround to this issue that I did for the same purpose.
I inserted the **hover** image as a **normal background** (in order to load) and then I went under the background overlay settings and placed the normal / hover images as the should be (I also set the background color on both states to be the same color as the background of my page). Be sure to have opacity set to 1 for the background overlay on both normal and hover states.
